### PR TITLE
PredefinedColorPalettes: Add Rekordbox Hotcue Color Palettes

### PR DESCRIPTION
**Pull request by Swiftb0y**
Supersedes #3743
Due to the "truncation" of the colors, some position get assigned the same color twice, highlighting the issue I tried to fix in #2902.
I may pick that up again now.

### CODE REVIEW

File: src/util/color/predefinedcolorpalettes.cpp [hunk 1]
```diff
@@ -203,6 +221,54 @@ const ColorPalette PredefinedColorPalettes::kSeratoDJProHotcueColorPalette =
...
+        kRekordboxHotcueColor14,
+        kRekordboxHotcueColor15,
+        kRekordboxHotcueColor16,
+};
+
+const ColorPalette PredefinedColorPalettes::kRekordboxCDJHotcueColorPalette =
+        ColorPalette(
+                QStringLiteral("Rekordbox CDJ Hotcue Colors"),
+                kRekordboxColorsSelection,
+                {9});
```
Comment (Holzhaus) [inline src/util/color/predefinedcolorpalettes.cpp:252]: Do we really need this palette if you can just select green as default hotcue color in the preferences?

**Review by Holzhaus — COMMENTED** [review]
Thanks.

File: src/util/color/predefinedcolorpalettes.cpp (hunk 1 above)
Comment (Swiftb0y) [reply src/util/color/predefinedcolorpalettes.cpp:252]: Good question, I kept it for the sake of completeness, but we can get rid of it if we document that in the preferences.

Comment (Holzhaus) [reply src/util/color/predefinedcolorpalettes.cpp:252]: Opinions? @ronso0 @daschuer

Comment (ronso0) [reply src/util/color/predefinedcolorpalettes.cpp:252]: yeah, let's drop it. we don't need to copy that 1:1, and if that green is vital for ex-RekordBox users ; ) they can easily pick it as default color.

**Comment by Swiftb0y** [comment]
Friendly ping  / request to merge?

**Comment by Holzhaus** [comment]
I cannot test this due to https://bugs.launchpad.net/mixxx/+bug/1921549 but the code looks ok and CI passes. LGTM, thank you!

**Comment by Holzhaus** [comment]
Yikes, I was too quick. This should have went into 2.3, not main. Can you open a new PR?

**Comment by Swiftb0y** [comment]
Oh, yes, sorry, my fault I didn't notice that as well. #3749

**Pull request merged**